src/ngraph/runtime/cpu/builder/quantized_conv.cpp

```diff
@@ -67,7 +68,49 @@ namespace ngraph
                     throw ngraph_error("unsupported parameters for QuantizedConvolution via DEX");
                 }
             }
+
+            template <>
+            void Builder::BUILDER_DECL(ngraph::op::QuantizedConvolutionRelu)
+            {
+                if (runtime::cpu::mkldnn_utils::use_mkldnn_kernel(node))
+                {
+                    auto qconvolution_relu =
+                        static_cast<const ngraph::op::QuantizedConvolutionRelu*>(node);
+                    auto& functors = external_function->get_functors();
+                    auto& arg0_tensor = external_function->get_tensor_data(args[0].get_name());
+                    auto& arg1_tensor = external_function->get_tensor_data(args[1].get_name());
+                    auto& out_tensor = external_function->get_tensor_data(out[0].get_name());
+                    auto& out1_tensor = external_function->get_tensor_data(out[1].get_name());
+                    auto& out2_tensor = external_function->get_tensor_data(out[2].get_name());
+
+                    auto& mkldnn_emitter = external_function->get_mkldnn_emitter();
+
+                    auto conv_index =
+                        mkldnn_emitter->build_convolution<ngraph::op::QuantizedConvolutionRelu>(
+                            node, args, out);
+                    auto& deps = mkldnn_emitter->get_primitive_deps(conv_index);
+                    float min_freezed_output = qconvolution_relu->get_freezed_output_min();
```

Comment:
trying to understand, why is it called `freezed_output_min()` and `freezed_output_max()` and not just output_min/max?

Reply:
Just because we do some graph transforms including freezing the graph and extracting values logged during training.